OrbitGl/GpuTrack.cpp

```diff
@@ -181,6 +200,9 @@ void GpuTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick) {
       } else {
         auto type = PickingID::LINE;
         batcher->AddVerticalLine(pos, size[1], z, color, type, &text_box);
+        // For lines, we can ignore the entire pixel into which this event falls.
+        min_ignore = time_graph_->GetTickFromWorld(std::floor(pos[0]));
```

Comment:
I suspect this is not quite correct. The idea is that min_ignore, max_ignore spans the timerange that is already covered by the vertical line drawn. But that depends on the thickness of the line drawn and how the line coordinates exactly are translated when drawing (pos is fractional).

Reply:
I think that conceptually this makes total sense as the line width is 1 pixel. I'm not sure we should be using pos[0] however, it is a floating point value (as opposed to double for "start_us" for example), and contains error from its own computation.  If we had the timegraph's min and max values expressed in "ticks" we could do something like:

min_ignore = min_timegraph_tick + ((timer.m_Start - min_timegraph_tick)/pixel_delta_in_ticks)* pixel_delta_in_ticks;

Or in a similar computation as above, we could use the start_us and the existing timegraph's min/max expressed in us, and then convert to ticks.

Reply:
Done.